submissions/aymkin/js-dom/index.js

```diff
@@ -0,0 +1,126 @@
+  },
+  {
+    name: "Grape",
+    imgSrc: "./img/grape.png",
+    description:
+      "A grape is a fruit, botanically a berry, of the deciduous woody vines of the flowering plant genus Vitis.\n\nGrapes can be eaten fresh as table grapes or they can be used for making wine, jam, juice, jelly, grape seed extract, raisins, vinegar, and grape seed oil. Grapes are a non-climacteric type of fruit, generally occurring in clusters."
+  },
+  {
+    name: "Pear",
+    imgSrc: "./img/pear.png",
+    description:
+      "The pear (/ˈpɛər/) tree and shrub are a species of genus Pyrus /ˈpaɪrəs/, in the family Rosaceae, bearing the pomaceous fruit of the same name. Several species of pear are valued for their edible fruit and juices while others are cultivated as trees."
+  },
+  {
+    name: "Durian",
+    imgSrc: "./img/durian.png",
+    description:
+      "The durian (/ˈdjʊəriən, ˈdʊr-, -æn/)[2] is the fruit of several tree species belonging to the genus Durio. There are 30 recognised Durio species, at least nine of which produce edible fruit, with over 100 named varieties in Indonesia,[3] 300 in Thailand and 100 in Malaysia.[4] Durio zibethinus is the only species available in the international market: other species are sold in their local regions."
+  }
+];
+
+const doc = document;
+
+const container = doc.createElement("div");
+container.setAttribute("class", "container");
+
+const pageName = doc.createElement("h1");
+pageName.setAttribute("id", "page-name");
+pageName.textContent = "JS DOM";
+
+const dictionaryName = doc.createElement("h2");
+dictionaryName.setAttribute("id", "dictionary-name");
+dictionaryName.textContent = "Fruit dictionary";
+
+const divMain = doc.createElement("div");
+divMain.setAttribute("class", "main");
+
+const divMenu = doc.createElement("div");
+divMenu.setAttribute("class", "menu");
+
+const navigation = doc.createElement("ul");
+navigation.setAttribute("id", "navigation");
+const nav = doc.querySelectorAll("li");
+
+for (let i = 1; i <= fruits.length; i += 1) {
+  const li = doc.createElement("li");
+  li.setAttribute("class", "list-element");
+  li.textContent = fruits[i - 1].name;
+  if (i - 1 === 0) {
+    li.setAttribute("class", "list-element clicked");
+  }
+  navigation.appendChild(li);
+}
+
+const divImage = doc.createElement("div");
+divImage.setAttribute("class", "image");
+
+const image = doc.createElement("img");
+image.setAttribute("id", "image");
+image.setAttribute("src", fruits[0].imgSrc);
+
+const divDescription = doc.createElement("div");
+divDescription.setAttribute("class", "description");
+
+const descriptionHeader = doc.createElement("h3");
+descriptionHeader.textContent = fruits[0].name;
+
+const descriptionParah = doc.createElement("p");
+descriptionParah.textContent = fruits[0].description;
+
```

Comment:
In my opinion, it is better to first declare all the constants and then proceeds to add/change properties

Reply:
Some const definition are made by loop, if I move it to the top, the code does not work properly
http://i.imgur.com/cAg6R17.png